### PR TITLE
feat: support for array as event parameter in useEventListener

### PR DESCRIPTION
**Pull request by askwuxue**
[[中文版模板 / Chinese template](https://github.com/alibaba/hooks/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/alibaba/hooks/issues/2434

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   support for array as event parameter in useEventListener   |
| 🇨🇳 Chinese |  useEventListener 事件入参支持数组         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: packages/hooks/src/useEventListener/index.ts [hunk 1]
```diff
@@ -62,15 +62,21 @@ function useEventListener(eventName: string, handler: noop, options: Options = {
         return handlerRef.current(event);
       };
 
-      targetElement.addEventListener(eventName, eventListener, {
-        capture: options.capture,
-        once: options.once,
-        passive: options.passive,
+      const eventNameArray = Array.isArray(eventName) ? eventName : [eventName];
+
+      eventNameArray.forEach((eventName) => {
```
Comment (crazylxr) [inline packages/hooks/src/useEventListener/index.ts]: 这个 eventName 和外部的 eventName 名字一样了，编辑器会有警告的，这个改一下吧

**Review by crazylxr — COMMENTED** [review]
另外再加一个使用例子

**Comment by askwuxue** [comment]
> 另外再加一个使用例子

已补充

File: packages/hooks/src/useEventListener/index.ts (hunk 1 above)
Comment (askwuxue) [reply packages/hooks/src/useEventListener/index.ts]: 好的

**Comment by crazylxr** [comment]
LGTM

**Pull request merged**